code/src/album.css

```diff
@@ -0,0 +1,122 @@
+
+:root {
+  --primary-headings-font-family: "Helvetica", sans serif;
+  --secondary-headings-font-family: "Helvetica", sans serif;
+  --paragraph-font-family: "Helvetica", sans serif;
+}
+
+/*ALBUM*/
+
+.album-name {
+  font-family: var(--secondary-headings-font-family);
+  font-size: 14px;
+  color: #fff;
+  text-decoration: none;
+}
+
+.album {
+  margin-bottom: 1.5rem;
+}
+
+.album-name:hover {
+  text-decoration: underline;
+}
+
+.album-cover {
+  width: 100%;
+  height: 100%;
+}
+
+/*album-cover gets opacity at hover on its parent element cover-and-icons*/
+
+.cover-and-icons:hover .album-cover {
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+.album-container {
+  place-items: center;
+}
+
+/*BUTTONS*/ 
+
+.cover-and-icons  {
+    position: relative;
+}
+
+/*img-overlay is displayed in block at hover on its parent element cover-and-icons. This makes the icons visible at hover on the album*/
+
+.cover-and-icons:hover .img-overlay {
+    display: block;
+}
+
+.img-overlay {
+    justify-content: center;
+    align-items: center;
+    display: none;
+}
+
+.icons-container {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 1;
+}
+
+.heart-icon {
+    margin-right: 30px;
+    filter: invert(0.8); /*this changes the color from black to white on the icons, 1 means that the icon will be fully white and a lower value is more opaque*/
+    width: 1.5rem;
+}
+
+.heart-icon:hover {
+    transition: 0.5s ease;
+    cursor: pointer;
+}
+
+.play-icon {
+    margin-right: 30px;
+    filter: invert(0.8); 
+    width: 3.5rem;
+}
+
+.play-icon:hover {
+    transform: scale(1.3);
+    transition: 0.5s ease;
+    cursor: pointer;
+    filter: invert(1); 
+}
+
+.info-icon {
+    margin-right: 20px;
+    filter: invert(0.8);
+    width: 1.5rem;
+}
+
+.info-icon:hover {
+    transition: 0.5s ease;
+    cursor: pointer;
+} 
+
+/*RESPONSIVENESS*/
+
+/*tablet*/
+@media ((min-width: 668px) and (max-width: 1023px)) {
+  .album-container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+/*desktop*/
+@media (min-width: 1024px) {
+  .album-container {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
```

Comment:
Thanks for this - this was the solution we were looking for but couldn't remember so we repeated ourselves a little bit here in our own project.
(We wrote 1fr 1fr etc)
/Ylva & Emma